services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/model/ReadBatch.java

```diff
@@ -37,14 +45,26 @@ private ReadBatch(BuilderImpl<?> builder) {
         this.keysAndAttributes = generateKeysAndAttributes(builder.requests, builder.mappedTableResource);
     }
 
+    /**
+     * All objects must be constructed using a Builder.
+     * @param itemClass to determine the parameterized type T of the builder
```

Comment:
Can we make this more user-friendly and less developer-centric. Something like: @param itemClass the class that items in this table map to

Reply:
Re-worded.